bicycleparameters/app-data/Crescendo/RawData/CrescendoMeasured.txt

```diff
@@ -0,0 +1,50 @@
+lF = 0.301+/-0.001
```

Comment:
I don't think you need these "Measured" files for the app.

Reply:
Okay. I went ahead and locally deleted all the RawData folders as well. They don't contain anything else.